### ISSUE TITLE
simplelogin.io blocked by nextDNS Threat Intelligence Feeds

### ISSUE THREAD

**Issue by zanona** [simple-login/website#20]
Apparently, during the last few days simplelogin.io somehow got into the Threat Intelligence Feeds on NextDNS, blocking request for everyone with that feed enabled, which seems a little unfair IMO. Perhaps it's worth reaching out to next DNS and request whitelisting the SimpleLogin?

![screenshot2](https://user-images.githubusercontent.com/32017/160230176-89e1ff0e-4ad5-4467-8da8-d6d85b012268.png)
![screenshot](https://user-images.githubusercontent.com/32017/160230179-17be1f91-ee69-4375-87f6-9e85cab796d8.png)

**Comment by nguyenkims** [comment]
@zanona thanks for letting us know. We'll try to get in touch with NextDNS so they can solve this issue.

**Comment by umarbutler** [comment]
@nguyenkims It happened again. I've flagged on their forum here https://help.nextdns.io/t/h7ylhvy/app-simplelogin-io-is-blocked

**Comment by Ncage1974** [comment]
Yep this is wierd. Been using nextdns for a long time & first time i've had a very legitimate domain blocked like this.. FYI @nguyenkims

**Comment by nguyenkims** [comment]
Thanks, we'll reach out to nextDNS. In the meantime, the workaround is to turn off threat intelligence in nextDNS.
